bitnami/orangehrm/templates/networkpolicy-backend-ingress.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if and .Values.networkPolicy.enabled .Values.networkPolicy.ingressRules.backendOnlyAccessibleByFrontend }}
+apiVersion: {{ include "common.capabilities.networkPolicy.apiVersion" . }}
+kind: NetworkPolicy
+metadata:
+  name: {{ printf "%s-backend" (include "common.names.fullname" .) }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  podSelector:
+    matchLabels:
+      {{- if .Values.networkPolicy.ingressRules.customBackendSelector }}
+      {{- include "common.tplvalues.render" (dict "value" .Values.networkPolicy.ingressRules.customBackendSelector "context" $) | nindent 6 }}
+      {{- else }}
+      app.kubernetes.io/name: mariadb
```

Comment:
I think thisis wrong

Reply:
In these webapps charts we want to make the backend (mariadb in these four cases) only accessible by the frontend. We are hardcoding `app.kubernetes.io/name: mariadb` to select the mariadb pods because, unless mariadb is removed from the [dependencies](https://github.com/bitnami/charts/blob/17fd9b9819a1091ff83b9b466976a664f13b932b/bitnami/orangehrm/Chart.yaml#L5-L7), it will always be there.

Reply:
What would happen if `mariadb.enabled` is `false` ? Or if you use an already existing deployment of mariadb instead of the dependency?

Reply:
If `mariadb.enabled=false` and:
- You don't provide an already existing deployment of mariadb → Deployment fails, so here we are covered
- You provide an already existing deployment of mariadb → Then you have to use `customBackendSelector` and the `app.kubernetes.io/name: mariadb` is not added
https://github.com/bitnami/charts/blob/f203bf54e61927b999bd8391e3997f3d5b6b169d/bitnami/orangehrm/values.yaml#L737-L741

Reply:
👍